base_layer/wallet/src/transaction_service/handle.rs

```diff
@@ -52,7 +52,7 @@ pub enum TransactionServiceRequest {
     GetCompletedTransaction(TxId),
     GetAnyTransaction(TxId),
     SetBaseNodePublicKey(CommsPublicKey),
-    SendTransaction((CommsPublicKey, MicroTari, MicroTari, String)),
+    SendTransaction((CommsPublicKey, MicroTari, MicroTari, String, bool)),
```

Comment:
I disagree with this. Adding a boolean argument is almost always bad because its not readable. Just look at this line: With no other context what does that last argument do? You can't know. The other arguments at least tell you something about their nature. Furthermore look at all the changes you had to make to TestNet utils to support this.

I would just add a new descriptive member to this enum and make calls for one side payments a new API call.